src/Entity/SenatorialCandidateManagedArea.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Entity;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="senatorial_candidate_areas")
+ * @ORM\Entity
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class SenatorialCandidateManagedArea
+{
+    /**
+     * @var int|null
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     */
+    private $id;
+
+    /**
+     * @ORM\ManyToMany(targetEntity="App\Entity\ReferentTag")
+     * @ORM\JoinTable(
+     *     name="senatorial_candidate_areas_tags",
+     *     joinColumns={
+     *         @ORM\JoinColumn(name="senatorial_candidate_area_id", referencedColumnName="id", onDelete="CASCADE")
+     *     },
+     *     inverseJoinColumns={
+     *         @ORM\JoinColumn(name="referent_tag_id", referencedColumnName="id")
+     *     }
+     * )
+     */
+    private $departmentTags;
+
+    public function __construct()
+    {
+        $this->departmentTags = new ArrayCollection();
+    }
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function getDepartmentTags(): iterable
```

Comment:
what about return the `array` directly here and avoid to call `toArray()` each time?

Reply:
it allows us to benefits from methods like "isEmpty", "filter" etc... Do you want to still return the array from here ?